wykorzystanie_formatu_paczki_archiwalnej.rst

```diff
@@ -0,0 +1,96 @@
+Wykorzystanie formatu paczki archiwalnej jako formatu danych EZD
+================================================================
+
+Wstęp
+-----
+
+W tym dokumencie opisana jest propozycja użycia tzw. formatu paczki
+archiwalnej [#paczka_archiwalna]_ jako głównego `formatu danych EZD`_.
+
+W dokumencie używane są aliasy:
+
+* "FPA" oznacza "format paczki archiwalnej"
+* "FD EZD" oznacza "format danych EZD"
+
+Zarys
+-----
+
+Elementami metryki sprawy są pojedyncze dokumenty XML, każdy w FPA.
+Treść pism przechowywana jest w oddzielnym miejscu (nazwijmy je
+"skład pism"). Elementy metryki mogą zawierać odniesienia do pism.
+
+Aby takie rozwiązanie było poprawne niezbędne są zmiany w FPA lub w
+FD EZD. Nie wszystkie zmiany podane poniżej są niezbędne.
+
+Wskazanie elementu poprzedniego metryki
+---------------------------------------
+
+W FD EZD poprzedni element wskazywany jest z użyciem URI. W FPA nie
+występuje wskazanie elementu poprzedniego jako specjalne pole do tego
+przeznaczone. W FPA jest możliwość tworzenia relacji między dokumentami.
+
+W FPA można zrealizować wskazanie poprzedniego elementu metryki za
+pomocą relacji odpowiedniego typu. Takie rozwiązanie nie jest dobre,
+ponieważ trudno wymusić dokładnie jedno wskazanie poprzedniego elementu
+metryki.
+
+W FPA można dodać pole wskazujące poprzedni element metryki.
+
+Wskazanie wykonawcy czynności
+-----------------------------
+
+W FD EZD wykonawca czynności jest wskazywany za pomocą URI, w FPA za
+pomocą pola :code:`tworca` typu :code:`Tworca`. Złożony typ
+:code:`Tworca` zawiera pole :code:`id` typu :code:`Id` (niepusty napis).
+
+W FD EZD można dopuścić wskazanie wykonawcy czynności za pomocą elementu
+typu :code:`Twórca`.
+
+W FPA można załączać URI wykonawcy w polu :code:`id` pola :code:`tworca`
+lub wskazywać wykonawcę jako URI (np. jako odnośnik do części tego
+samego dokumentu).
+
+Specyfikacja czynności
+----------------------
+
+W FD EZD specyfikacja czynności jest polimorficznym polem. W FPA na
+specyfikację czynności składa się wiele pól. Dużo pól określa metadane,
+a zatem zmiana (także nadanie) metadanych jest reprezentowane przez
+stworzenie dokumentu z odpowiednimi wartościami tych pól. W polu
```

Comment:
Zdaje się, że pliki XML w FPA są wiązane z dokumentami po nazwie pliku. Jeśli jakiś dokument miał dwukrotnie nadawane metadane (i ma, jak rozumiem, utworzone dwa pliki XML), to w jakich sposób dokona się to powiązanie?